contrib/tiflash-proxy-cmake/CMakeLists.txt

```
@@ -18,10 +20,12 @@ if(TIFLASH_LLVM_TOOLCHAIN AND USE_LIBCXX)
     if(LINKER_NAME)
         set(TIFLASH_RUSTFLAGS "-C link-arg=-fuse-ld=${LINKER_NAME} ${TIFLASH_RUSTFLAGS}")
     endif()
-    set(TIFLASH_RUST_ENV CXXSTDLIB=c++ CMAKE=${CMAKE_COMMAND} CFLAGS=-w CXXFLAGS=-w RUSTFLAGS=${TIFLASH_RUSTFLAGS} PROTOC=${Protobuf_PROTOC_EXECUTABLE} PROTOC_INCLUDE=${Protobuf_INCLUDE_DIR})
-    message(STATUS "enforce LLVM toolchain for rust: ${TIFLASH_RUST_ENV}")
+    set(TIFLASH_RUST_ENV CXXSTDLIB=c++ ${TIFLASH_RUST_ENV} RUSTFLAGS=${TIFLASH_RUSTFLAGS} PROTOC=${Protobuf_PROTOC_EXECUTABLE} PROTOC_INCLUDE=${Protobuf_INCLUDE_DIR})
```

```suggestion
    set(TIFLASH_RUST_ENV CC=${CMAKE_C_COMPILER} CXX=${CMAKE_CXX_COMPILER} CXXSTDLIB=c++ ${TIFLASH_RUST_ENV} RUSTFLAGS=${TIFLASH_RUSTFLAGS} PROTOC=${Protobuf_PROTOC_EXECUTABLE} PROTOC_INCLUDE=${Protobuf_INCLUDE_DIR})
```

How about using same compiler like tiflash? @SchrodingerZhu 
If not set, proxy may use default cpp compiler `cc (GCC) 4.8.5`.

---

it is surely a better idea

---

we can also set CMAKE to be the current executable.

---

maybe we can use sth like $CMAKE_ROOT/bin/cmake. not quite sure

---

for release pipeline, CXX/CC is set globally.